src/CatBox.NET/Client/CatBox/CatBox.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CatBox.NET.Requests.CatBox;
+
+namespace CatBox.NET.Client;
+
+/// <inheritdoc/>
+public sealed class Catbox : ICatBox
+{
+    private readonly ICatBoxClient _client;
+
+    /// <summary>
+    /// Instantiate a new catbox class 
+    /// </summary>
+    /// <param name="client">The CatBox Api Client</param>
+    public Catbox(ICatBoxClient client)
+    {
+        _client = client;
+    }
```

Comment:
I've noticed you did this in other places, but
```cs
/// <param name="client">The CatBox Api Client (<see cref="ICatBoxClient"/>)</param>
```
If you wanted to be more consistent. That or drop "The CatBox Api Client" and only have the reference pointing to `ICatBoxClient`.

Reply:
Done